tests/tasks/dbt/test_dbt.py

```diff
@@ -3,12 +3,16 @@
 from prefect.tasks.dbt.dbt_cloud_utils import TriggerDbtCloudRunFailed
 
 import responses
+from responses import matchers
 import pytest
 
+import prefect
 from prefect import Flow
 from prefect.engine.signals import FAIL
 from prefect.tasks.dbt import DbtShellTask, DbtCloudRunJob
 
+USER_AGENT_HEADER = {"user-agent": "prefect-{v}".format(v=prefect.__version__)}
```

Comment:
I wonder if this can be imported from src/prefect/tasks/dbt/dbt_cloud_utils.py?

Reply:
I agree it would be nice to just import this

Reply:
`USER_AGENT_HEADER` is now imported into the tests. I'm using a `header_matcher` with responses to check for the user-agent header in requests during unit tests. The tests will fail if the expected user agent header isn't present.

Reply:
Ah but if we're importing it and someone changes `USER_AGENT_HEADER` then your tests might pass. It might be worth including a test that the `USER_AGENT_HEADER` constant includes "prefect" since that's its purpose. I don't feel strongly though.